src/utils/showError.ts

```diff
@@ -0,0 +1,19 @@
+import { ShowError } from '../types/ShowErrors';
+
+export const showErrors = (error: ShowError) => {
```

Comment:
seems redundant
Do you use it anywhere? I guess you may delete it when adding to the portfolio (you will create a clear repo, not forked and w/o the mentor's comments)

Reply:
You are right